### PR TITLE
English tweaks to language of RAWcooked website

### PR DESCRIPTION
**Pull request by digitensions**
A few amendments, as well as updates to project status which currently has it as still 'under development'. I've assumed this incorrect following the launch at NTTW3, but feel free to correct. Also, was unsure of the language surround the 10000€ funding passage (from line 132), so I've edited it how I think it should read, but this may be completely wrong!  I've used english spellings, but happy to change 'flavour to flavor' if preferred.  I assume that the links remain intact/current, but I've not tested them, they could require updates.  Happy if anything is useful for you both :)

### CODE REVIEW

**Comment by JeromeMartinez** [comment]
> still 'under development'

Oops... Right, now we have a stable version!

> I've used english spellings

I use to have US-English, so I think I prefer to keep it for the moment, if it is not too much weird.

**Comment by digitensions** [comment]
No not weird at all. Can I make edits to the pull request still?

**Comment by JeromeMartinez** [comment]
> Can I make edits to the pull request still?

Yes. When you add a commit to your branch, it is automatically updated here.

(more comments to come, maybe good to wait for the review feedback before a new commit)

File: src/RAWcookedBundle/Resources/views/Default/index.html.twig [hunk 1]
```diff
@@ -106,40 +106,40 @@
...
-<p>This way the contributing bodies and individuals are sure their files can be processed by the software.</p>
-<p>The price is 1&nbsp;000&nbsp;&euro; for basic support (bug fixes) and one non-free flavour, then 500&nbsp;&euro; for each following non-free flavour (additional flavours can be added later if you are not sure of how many flavours you need now, at the same price).</p>
+<p>MediaArea.net has achieved implementation of RAWcooked’s basic module with thanks to AV Preservation reto.ch, who has guaranteed software development with funding of 10,000€.</p>
+<p>The archive community or interested industry users are invited to <a href="{{ path('ma_contact') }}">buy</a> additional flavours required by their institution - such as components, bit depths, endianess, packing, etc.  Required flavours will be fully tested by MediaArea.net to exact specification before being implemented into the software.</p>
```
Comment (JeromeMartinez) [inline src/RAWcookedBundle/Resources/views/Default/index.html.twig]: Actually they are already implemented.
So misleading.
Something like:
"The archive community or interested industry users are invited to \<a href="{{ path('ma_contact') }}">buy\</a> additional flavours required by their institution - such as components, bit depths, endianess, packing, etc in order to help the long term sustainability of the project and further development ."

File: src/RAWcookedBundle/Resources/views/Default/index.html.twig [hunk 2]
```diff
@@ -106,40 +106,40 @@
...
+<p>This model allows contributors the assurance that their files will work seamlessly with the software from the outset.</p>
+<p>The price is 1000€ for basic support including bug fixes and one additional flavour, then 500 € for each following flavour.  Additional flavours can be added at a later if you are not sure of how many flavours you need immediately.</p>
```
Comment (JeromeMartinez) [inline src/RAWcookedBundle/Resources/views/Default/index.html.twig]: "at a later" seems weird, wanted?

File: src/RAWcookedBundle/Resources/views/Default/index.html.twig [hunk 3]
```diff
@@ -106,40 +106,40 @@
...
-Priority is given to formats and flavors from our sponsors. You can test with this <a href="https://github.com/MediaArea/RAWcooked-RegressionTestingFiles/blob/master/Features/AV_Package/818_DCDM_P3_IA_FIC_000918/818_OV/818_OV_0086945.dpx?raw=true">file</a> (or a <a href="https://github.com/MediaArea/RAWcooked-RegressionTestingFiles/tree/master/Features/AV_Package/818_DCDM_P3_IA_FIC_000918/818_OV">sequence of files</a>).</p>
+<p>RAWcooked had its first official release October 2018, with support for a limited number of DPX flavours. To use RAWcooked simply type  <code>rawcooked /RAWsequenceDPX/</code> which begins the encoding using FFmpeg to create an MKV encoded file in the same folder, leaving the original DPX sequence in tact.  You can also use this command <code>rawcooked yourdpx.dpx</code> using the first DPX file in a sequence within your folder. To return the MKV back to its RAW source format use <code>rawcooked yourdpx.mkv</code> which uses RAWcooked software to unpack your RAW sequence bit-by-bit, including all original metadata and sidecar files.<br>
```
Comment (JeromeMartinez) [inline src/RAWcookedBundle/Resources/views/Default/index.html.twig]: "in tact" seems weird, wanted?

**Review by JeromeMartinez — CHANGES_REQUESTED** [review]
Thanks for the better English!
Some remarks / change requests.

**Comment by JeromeMartinez** [comment]
@retokromer some comments?

File: src/RAWcookedBundle/Resources/views/Default/index.html.twig (hunk 3 above)
Comment (digitensions) [reply src/RAWcookedBundle/Resources/views/Default/index.html.twig]: It should read 'intact', sorry! Meaning, not damaged or impaired in any way, complete.

**Comment by retokromer** [comment]
@JeromeMartinez I’m commuting and cannot read well on cell… I’ll delve into it a little later on a smarter computer.

**Comment by digitensions** [comment]
I'll wait for Reto's before I do anything if that's okay? Thanks for letting me play at pull requests with such a precious resource :)

File: src/RAWcookedBundle/Resources/views/Default/index.html.twig [hunk 4]
```diff
@@ -17,8 +17,8 @@
...
-                When needed, the uncompressed source is retrieved <b>bit-by-bit</b>, in a manner faster than uncompressed sources directly stored on LTO cartridges.
+                <strong>RAWcooked</strong>  easily encodes RAW audio-visual image sequences into a lossless video stream, reducing the file size by between one and two thirds.  Using FFmpeg the audio-visual data is encoded into a Matroska container (MKV) using the video codec FFV1, and audio codec FLAC. The metadata accompanying the RAW data is fully preserved, along with additional sidecar files such as MD5 checksums, LUT or XML if desired. This allows for the management of these audio-visual file formats in an effective and transparent way. The lossless Matroska video stream can be played in VLC or MPV media players, and writing and retrieving from storage devices such as LTO is significantly quicker.<br>
```
[the remark this answers is not in the document]
Comment (retokromer) [reply src/RAWcookedBundle/Resources/views/Default/index.html.twig]: There is also a not needed double space at `</strong> easily`

[the remark this answers is not in the document]
Comment (retokromer) [reply src/RAWcookedBundle/Resources/views/Default/index.html.twig]: At `thirds. Using` as well.

File: src/RAWcookedBundle/Resources/views/Default/index.html.twig (hunk 2 above)
Comment (retokromer) [reply src/RAWcookedBundle/Resources/views/Default/index.html.twig]: Unify either with space or without space between the amount and the currency.

Comment (retokromer) [reply src/RAWcookedBundle/Resources/views/Default/index.html.twig]: There is a double space at `flavour.  Additional`.

File: src/RAWcookedBundle/Resources/views/Default/index.html.twig (hunk 3 above)
Comment (retokromer) [reply src/RAWcookedBundle/Resources/views/Default/index.html.twig]: Double spaces at `type <code>rawcooked` and `tact. You`.

**Review by retokromer — COMMENTED** [review]
A few nits. Yet sound much better to me (not fluent in English). Thank you very much, @b00giehead!

**Review by retokromer — APPROVED** [review]

**Review by retokromer — APPROVED** [review]
Good catch!

**Review by retokromer — APPROVED** [review]
I always prefer a space between the amount and the unit!

**Review by JeromeMartinez — APPROVED** [review]

**Comment by JeromeMartinez** [comment]
Main issue was a misunderstanding about how to put HTML tags (no quotation marks ;-) ) + 1 missing semi-column in my review. @digitensions check https://github.com/MediaArea/MediaArea-Website/pull/257/commits/fa6540e373d70dc01634d7368858c1ec724fd78f for how to write HTML tags.

This is online, thank you @digitensions for the update, very useful (no more weird English)!

**Pull request merged**